.github/workflows/configlet.yml

```diff
@@ -13,5 +13,3 @@ permissions:
 jobs:
   configlet:
     uses: exercism/github-actions/.github/workflows/configlet.yml@main
-    with:
-      fmt: true
```

Comment:
You still want this to be enabled, right?

Reply:
Yes, but with this change reverted. Otherwise, once you merge this PR, you'll get a new PR to re-add these lines (it's what the config file does).